lambdas/evaluate-gpg45-scores/src/test/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandlerTest.java

```diff
@@ -308,6 +281,9 @@ M1A, new Gpg45ScoresDto(List.of(new EvidenceDto(4, 2)), 0, 0, 2)),
                                 M1B, new Gpg45ScoresDto(List.of(new EvidenceDto(3, 2)), 1, 0, 2))),
                 ipvSessionItem.getRequiredGpg45Scores());
 
+        InOrder inOrder = inOrder(ipvSessionItem, ipvSessionService);
+        inOrder.verify(ipvSessionService).updateIpvSession(ipvSessionItem);
+        inOrder.verify(ipvSessionItem, never()).setVot(any());
```

Comment:
I don't think you need this here. My original comment was around if you wanted to check that the required scores had been stored in the session (that's what this test is testing). We're already checking that the VOT isn't set in the lines just below.

If you're not worried about testing that the required scores have been written correctly I probably wouldn't change this test at all. Considering what your PR is for, you probably don't need to be updating this test at all.

Reply:
Done.

Reply:
When we ran the feature test together with test team, we encountered some errors, so we had to take the latest changes you made in the main branch and changes some unit tests. I have reverted this test to its previous state.